ntbs-service/Jobs/DataQualityAlertsJob.cs

```diff
@@ -34,63 +34,71 @@ public async Task Run(IJobCancellationToken token)
                 await _alertService.AddUniqueAlertAsync(alert);
             }
-
+            
             var notificationsForClinicalDatesAlerts =
                 await _dataQualityRepository.GetNotificationsEligibleForDataQualityClinicalDatesAlerts();
             foreach (var notification in notificationsForClinicalDatesAlerts)
             {
                 var alert = new DataQualityClinicalDatesAlert {NotificationId = notification.NotificationId};
                 await _alertService.AddUniqueAlertAsync(alert);
             }
-
+            
             var notificationsForClusterAlerts =
                 await _dataQualityRepository.GetNotificationsEligibleForDataQualityClusterAlerts();
             foreach (var notification in notificationsForClusterAlerts)
             {
                 var alert = new DataQualityClusterAlert {NotificationId = notification.NotificationId};
                 await _alertService.AddUniqueAlertAsync(alert);
             }
-
+            
             var notificationsForTreatmentOutcome12Alerts =
                 await _dataQualityRepository.GetNotificationsEligibleForDataQualityTreatmentOutcome12Alerts();
             foreach (var notification in notificationsForTreatmentOutcome12Alerts)
             {
                 var alert = new DataQualityTreatmentOutcome12 {NotificationId = notification.NotificationId};
                 await _alertService.AddUniqueAlertAsync(alert);
             }
-
+            
             var notificationsForTreatmentOutcome24Alerts =
                 await _dataQualityRepository.GetNotificationsEligibleForDataQualityTreatmentOutcome24Alerts();
             foreach (var notification in notificationsForTreatmentOutcome24Alerts)
             {
                 var alert = new DataQualityTreatmentOutcome24 {NotificationId = notification.NotificationId};
                 await _alertService.AddUniqueAlertAsync(alert);
             }
-
+            
             var notificationsForTreatmentOutcome36Alerts =
                 await _dataQualityRepository.GetNotificationsEligibleForDataQualityTreatmentOutcome36Alerts();
             foreach (var notification in notificationsForTreatmentOutcome36Alerts)
             {
                 var alert = new DataQualityTreatmentOutcome36 {NotificationId = notification.NotificationId};
                 await _alertService.AddUniqueAlertAsync(alert);
             }
-
+            
             var notificationsForDotVotAlerts =
                 await _dataQualityRepository.GetNotificationsEligibleForDotVotAlerts();
-            foreach (var notification in notificationsForDotVotAlerts) 
+            foreach (var notification in notificationsForDotVotAlerts)
             {
                 var alert = new DataQualityDotVotAlert {NotificationId = notification.NotificationId};
                 await _alertService.AddUniqueAlertAsync(alert);
             }
 
+            var possibleDuplicateNotifications =
```

Comment:
I've added a comment to the data quality job performance ticket to highlight that everywhere else we are grabbing more than just the ids when all we are using are the ids and we populate the alerts with tbservicecode etc. later on in AddUniqueAlertAsync so it might be something to look at there

Reply:
nice, i like that!